Uplift/Views/DropdownView.swift

```diff
@@ -49,6 +51,7 @@ class DropdownView: UIView {
          headerViewHeight: CGFloat,
          contentView: UIView,
          contentViewHeight: CGFloat,
+         arrowImageView: UIImageView? = nil,
```

Comment:
why are we making this optional? 
on another note, I might be wrong the arrowImageView is the same for all drop-down cells right? we shouldn't need to make this an argument and just use the same arrowImageView for every `DropdownView` cc @ywsang

Reply:
All facility dropdown cells have the same arrow, but this dropdown view is not meant exclusively for the facility cells. Setting the arrow as an optional argument allows creation of dropdown views with different kinds of arrows, or no arrows at all. For example, the dropdown in `FilterViewController` (which I have not yet replaced with this new kind of dropdown view) uses a different kind of arrow.

Reply:
oh i see ok my bad i thought we were only using it for facilities. in this case, I think that the `arrowImageView` should just be handled by the `headerView` that is already passed in rather than inside the `DropdownView` itself. i think this makes more sense because the arrow image view is a part of the `headerView` anyways.